blueocean-web/gulpfile.js

```diff
@@ -55,36 +55,45 @@ builder.bundle('src/main/js/try.js')
     .import('jquery-detached', 'core-assets/jquery-detached:jquery2') // Bundled in Jenkins 2.x
     .less('src/main/less/try.less');
 
-//
-// Copy/link the JDL assests into the webapp dir, making them available at runtime.
-//
-var isWindows = /^win/.test(process.platform);
-var assetsDstPath = './src/main/webapp/assets';
-if (isWindows) {
-    var assestsCopyDone = false;
-    builder.onPreBundle(function() {
-        if (!assestsCopyDone) {
-            assestsCopyDone = true;
-            var ncp = require('ncp').ncp;
+// Copy/link library assests into the src/main/webapp/assets dir, making them available at runtime.
+linkAssets('jdl', '@jenkins-cd/design-language/dist/assets');
+linkAssets('corejs', '@jenkins-cd/blueocean-core-js/dist/assets');
 
-            // wipe the destination directory and recreate.
-            if (fs.existsSync(assetsDstPath)) {
-                rmdir(assetsDstPath);
-            }
-            fs.mkdirSync(assetsDstPath);
-            // copy assets from stc to dsy.
-            var assetsSrcPath = './node_modules/@jenkins-cd/design-language/dist/assets';
-            ncp(assetsSrcPath, assetsDstPath, function (err) {
-                if (err) {
-                    return logger.logError(err);
+/**
+ * Link (or copy) the specified module's subdir to a dir within /src/main/webapp/assets
+ * @param {string} dirName name of directory link
+ * @param {string} modulePath path within module, e.g. '@org-name/module/name/some/path
+ */
+function linkAssets(dirName, modulePath) {
+    var isWindows = /^win/.test(process.platform);
+    var assetsDstPath = './src/main/webapp/assets/' + dirName;
```

Comment:
:( I know this isn't a change, but not a fan of generating things into the source dir